core/types/rollup_cost.go

```diff
@@ -124,6 +124,11 @@ func NewL1CostFunc(config *params.ChainConfig, statedb StateGetter) L1CostFunc {
 	forBlock := ^uint64(0)
 	var cachedFunc l1CostFunc
 	selectFunc := func(blockTime uint64) l1CostFunc {
+		// Alfajores requires a custom cost function see link for a detailed explanation
+		// https://github.com/celo-org/op-geth/pull/271
+		if config.IsCel2(blockTime) && config.ChainID.Uint64() == params.CeloAlfajoresChainID {
```

Comment:
We'll want something like `&& !IsNextHardfork(blockTime)` when the next hard fork comes along. How do we make sure not to forget this? Maybe create an issue "Todos for next hardfork" where we can add these kind of things?